Sources/ContributeWordPress/Documentation.docc/Documentation.md

```diff
@@ -1,13 +1,362 @@
 # ``ContributeWordPress``
 
-<!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
+**ContributeWordPress** provides an easy to use API for importing a **WordPress** site into **Publish**.
 
 ## Overview
 
-<!--@START_MENU_TOKEN@-->Text<!--@END_MENU_TOKEN@-->
+![Logo](ContributeWordPressLogo-DocC.png)
+
+### Requirements 
+
+**Apple Platforms**
+
+- Xcode 14.3.1 or later
+- Swift 5.8 or later
+- iOS 14 / watchOS 7 / tvOS 14 / macOS 12 or later deployment targets
```

Comment:
Do we support platforms other than macOS?

Reply:
good point